Unknown variable 'default'

Hi!

I am currently trying out your knative example and added the following line to the Background section, because I currently do not have a broker running on my cluster.

`Background:
    Given Knative event consumer timeout is 20000 ms
    Given create Knative broker default
    Given Knative broker default is running
    Given variable id is "citrus:randomNumber(4)"`

I get the following exception on the step:
 '[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]   And send Knative event                                                                                # org.citrusframework.yaks.knative.SendEventSteps.createEvent(io.cucumber.datatable.DataTable)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       com.consol.citrus.exceptions.TestCaseFailedException: Unknown variable 'default'
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       at com.consol.citrus.DefaultTestCase.executeAction(DefaultTestCase.java:144)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       at com.consol.citrus.DefaultTestCaseRunner.run(DefaultTestCaseRunner.java:125)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       at org.citrusframework.yaks.knative.SendEventSteps.sendEventRequest(SendEventSteps.java:149)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       at org.citrusframework.yaks.knative.SendEventSteps.createEvent(SendEventSteps.java:121)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1]       at ✽.send Knative event(classpath:org/citrusframework/yaks/knative-eventing.feature:15)
[test-knative-eventing-c351hg2cjbf9cuksbe10-snsdn test-1] Caused by: com.consol.citrus.exceptions.CitrusRuntimeException: Unknown variable 'default''

The name of the variable "default" is the name of the namespace I run the test in.

Is this a bug or an application error from me?

--- a/java/steps/yaks-knative/src/main/java/org/citrusframework/yaks/knative/KnativeSettings.java
+++ b/java/steps/yaks-knative/src/main/java/org/citrusframework/yaks/knative/KnativeSettings.java
@@ -143,7 +143,7 @@ public class KnativeSettings {
     public static String getBrokerUrl() {
         String brokerUrlDefault = YaksSettings.isOpenshiftCluster() ?
                 String.format("http://%s", getBrokerHost()) :
-                String.format("http://%s/${%s}/${%s}", getBrokerHost(), getNamespace(), KnativeVariableNames.BROKER_NAME.value());
+                String.format("http://%s/%s/${%s}", getBrokerHost(), getNamespace(), KnativeVariableNames.BROKER_NAME.value());
         return System.getProperty(BROKER_URL_PROPERTY,
                 System.getenv(BROKER_URL_ENV) != null ? System.getenv(BROKER_URL_ENV) : brokerUrlDefault);
     }

